### ISSUE TITLE
Feature Request: change playback speed

### ISSUE THREAD

**Issue by KVAnton-WEB** [Sn8z/Poddr#172]
Please, Add a function to change the playback speed, for example, as in YouTube

Wonderful software, on Linux looks better than all competitors :)

**Comment by Sn8z** [comment]
@KVAnton-WEB Thank you very much :)
That's a great idea! I'm currently migrating Poddr to Angular 7 so it might take a while before I can add it but I will certainly do so 💃

**Comment by cverster** [comment]
I agree with this feature request! Big please!

**Comment by mizzunet** [comment]
Any update on this?

**Comment by Sn8z** [comment]
@mizzunet Yes, this is implemented in the development branch but haven't been released yet. Real life things and some other issues have delayed the new version for a while now but I will try to release it before the end of the year 🕺

**Comment by Sn8z** [comment]
@mizzunet 
> So Poddr, built out of dev branch will have this feature, right?

Yes, that's correct!
But the dev branch might not always be working correctly and can contain features or code that won't be released at all in the end.

**Comment by mizzunet** [comment]
> But the dev branch might not always be working correctly 

That's fine, I could be Poddr's tester (:
